modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -535,36 +533,6 @@ If you're running the SDK on your laptop against a Capella cluster, see further
 
 
 
-
```

Comment:
Could even delete some more of the whitespace lines above too, while you're at it 😂